cmd/influxd/config.go

```diff
@@ -98,6 +98,7 @@ type Config struct {
 	Logging struct {
 		File              string `toml:"file"`
 		WriteTraceEnabled bool   `toml:"write-tracing"`
+		RaftTraceEnabled  bool   `toml:"raft-tracing"`
```

Comment:
Should just remove `Enabled` from these. Otherwise everything in this struct is going to have that after it. Besides, this looks fine:

```go
if log.WriteTrace {
  // do stuff
}
```